src/ViewHelpers/CarViewHelper.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace KoalaCars\ViewHelpers;
+
+class CarViewHelper
+{
+    public static function displayAllCars(array $cars): string
+    {
+        $output = '';
+        foreach ($cars as $car) {
+            if (is_object($car)) {
```

Comment:
`is_object` just checks for any kind of object, we specifically need a `CarEntity`, why not create a private method that takes a `CarEntity` as a param and returns the HTML for 1 car, then call that method inside this foreach